lib/LoadingPane/LoadingPane.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  Icon,
+  Pane,
+} from '@folio/stripes/components';
+
+const LoadingPane = ({ onClose }) => (
+  <Pane
+    id="pane-loading"
+    defaultWidth="fill"
+    dismissible
+    onClose={onClose}
+  >
+    <div>
```

Comment:
For what you need this `div`?

Reply:
To make a container for future testing and positioning, because `Icon` it's a `<span>`